### DISCUSSION TITLE
Disk icons on the Desktop

### DISCUSSION THREAD

**Discussion by Lt-Ty** [elementary/triage#216] [Q&A]
I use elementary because it's VERY Mac-like. But I can't find a way to make my disk(s) appear on the desktop. (If it's a setting I've missed, please let me know.) On the Mac I can simply right-click the disk icon and "Get Info", to show me the size, and available space of the disk. I know this information can be retrieved via the bash Command Line, but that is very UN-Mac-like.

**Comment by danirabbit** [comment]
Things like files, folders, and disks are all available from the Files app. You can see any connected disks in the sidebar on the left side of the window. If you secondary-click one of these and select "Properties" you can see more specific disk usage information
